src/Toggle/ToggleChecker.php

```diff
@@ -9,6 +9,7 @@
 
 class ToggleChecker implements IsToggleActive
 {
+    const TOGGLE_TYPE_GROUP = 'group';
```

Comment:
I could, but this doesn't feel like the right place for this either. Why would the ToggleChecker hold the possible values for toggle types? :-/